windup.info.yml

```diff
@@ -0,0 +1,20 @@
+name: Windup
+description: 'A clean theme for Drupal 8'
+core: 8.x
+type: theme
+screenshot: screenshot.png
+stylesheets-remove:
+  - core/assets/vendor/normalize-css/normalize.css
+  - core/modules/system/css/system.theme.css
+  - core/modules/system/css/system.menus.css
+
+libraries:
+  - windup/global-styling
+#  - windup/global-js
```

Comment:
Yeah, the syntax seems to be different for JS (or at least I had issues when I copied windup/global-styling and tried to make it JS) AND the example windup.js is an alert... so I didn't want to include that out of the box.

Thoughts?